recipe/meta.yaml

```diff
@@ -57,21 +57,23 @@ requirements:
         - xerces-c
         - libnetcdf
         - krb5
-        - libtiff
         - libpng
-        - openjpeg
+        - openjpeg  # [not py35]
```

Comment:
Actually I just checked and there is no openjpeg in `defaults`, so let's leave it alone for now.